pkg/provider/github/status.go

```diff
@@ -87,6 +87,18 @@ func isPendingApprovalCheckrun(run *github.CheckRun) bool {
 	return false
 }
 
+func isFailedCheckrun(run *github.CheckRun) bool {
+	if run == nil || run.Output == nil {
+		return false
+	}
+	if run.Output.Title != nil && strings.Contains(*run.Output.Title, "Failed") &&
+		run.Output.Summary != nil &&
+		strings.Contains(*run.Output.Summary, "failed") {
```

Comment:
is it normal you have failure earlier and now failed?

Reply:
Yes `failure` is added initially in the conclusion and based on the conclusion Title and Summary has been setup 
https://github.com/openshift-pipelines/pipelines-as-code/blob/main/pkg/provider/github/status.go#L335-L337